lib/cli/transaction_str_params.rs

```diff
@@ -55,9 +55,15 @@ pub struct TransactionStrParams<'a> {
     /// Session args in json for use with the transaction
     pub session_args_json: &'a str,
     /// The pricing mode to use with the transaction
-    pub maybe_pricing_mode: Option<&'a str>,
+    pub pricing_mode: &'a str,
```

Comment:
Small detail do None for "classic" and Some("fixed") for the rest of options ?

Reply:
maybe_pricing_mode also would indicate Option in `pricing_mode: maybe_pricing_mode,`